README.md

```
@@ -111,6 +111,22 @@ docker run --rm -p 3000:3000 \
   service-atlas-frontend
 ```
 
+### Using Docker Compose
+
+1. Clone the repository
+2. Update .env file
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)